test/test_core.py

```diff
@@ -7709,7 +7709,7 @@ def test_webidl(self, mode, allow_memory_growth):
     # Export things on "TheModule". This matches the typical use pattern of
     # the bound library being used as Box2D.* or Ammo.*, and we cannot rely
     # on "Module" being always present (closure may remove it).
-    self.emcc_args += ['--post-js=glue.js', '--extern-post-js=extern-post.js']
+    self.emcc_args += ['-s', 'EXPORTED_FUNCTIONS=_malloc,_free', '-s', 'EXPORTED_RUNTIME_METHODS=stringToUTF8', '--post-js=glue.js', '--extern-post-js=extern-post.js']
```

Comment:
Can you make these into single arguments (e.g. `-sEXPORTED_FUNCTIONS`) rather than pairs?

Reply:
Sure, I guess that PR was created back when we used passed `-s` as a separate argument in our test code.